query-performance/query-tags.mdx

```diff
@@ -0,0 +1,71 @@
+---
+title: 'Query Tags'
+backlink_href: /docs/query-performance
+backlink_title: 'Query Performance'
+---
+
+Query tags are key-value pairs of runtime information of a query that you can
+add to the SQL statements as a query comment. The runtime information can be a
+name of the background job that is running the query, or a trace ID.
+
+Query tags help to find out where the query is coming from, or also help to find
+the corresponding query of the specific background job or a slow query.
```

Comment:
I think we could also add another example here:
```suggestion
Query tags can help determine where a query is coming from, support [tracing](https://pganalyze.com/docs/opentelemetry), or help track down the specific background job corresponding to a slow query.
```
Also, I don't quite follow the background job example, so I reworded that slightly.

Reply:
Ah, I see, you already link that below. I think it's still okay to use as an example, but having another example would be fine, too.

Reply:
Oh actually you're linking external docs below. It might be nice to link to our docs on OpenTelemetry somewhere (either an example here or somewhere else in this page), since otel is a big reason to use query tags.